openedx_webhooks/lib/edx_repo_tools_data/test/models/test_people.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pytest
+
+from openedx_webhooks.lib.exceptions import NotFoundError
+
+
+class TestGet:
```

Comment:
Test classes are just used for organization, right? In this case it makes sense to organize all the tests having to do with different scenarios of the same method. I'm anticipating that `People` will acquire more properties and methods.